searchtool/server/views/layout.hbs

```diff
@@ -183,7 +183,8 @@
                           {{#if term}}
                             <button aria-expanded="false" aria-haspopup="true" data-toggle="dropdown" class="btn btn-default dropdown-toggle" id="btn-pgActions">Download <span class="caret"></span></button>
                             <ul aria-labelledby="btn-pgActions" class="dropdown-menu">
-                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}">Download CSV</a></li>
+                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}&textdata=yes">CSV with text</a></li>
```

Comment:
@noonkay Oops.  I don't know why I used yes and no.  It was pretty late when I did that lol.  I changed it.  I think its fine to always keep both options available regardless of results.  If they have less than 2000 rows of results and still want to download without text, then we can just leave that up to them.